frontend/packages/console-app/src/redux/reducers/cloud-shell-selectors.ts

```diff
@@ -7,3 +10,8 @@ export const isCloudShellExpanded = (state: RootState): boolean =>
 
 export const isCloudShellActive = (state: RootState): boolean =>
   !!state.plugins?.console?.[cloudShellReducerName]?.isActive;
+
+export const getCloudShellCommand = (state: RootState): boolean =>
+  state.plugins?.console?.[cloudShellReducerName]?.command;
```

Comment:
You probably need to return `null` to match the correct signature:
```suggestion
export const getCloudShellCommand = (state: RootState): string | null =>
  state.plugins?.console?.[cloudShellReducerName]?.command ?? null;
```